drivers/volume/portworx/portworx.go

```diff
@@ -946,15 +946,6 @@ func (d *portworx) WaitDriverUpOnNode(n node.Node, timeout time.Duration) error
 	logrus.Debugf("waiting for PX node to be up: %s", n.Name)
 	t := func() (interface{}, bool, error) {
 		logrus.Debugf("Getting node info: %s", n.Name)
-		for _, addr := range n.Addresses {
-			err := d.testAndSetEndpointUsingNodeIP(addr)
-			if err != nil {
-				return "", true, &ErrFailedToWaitForPx{
-					Node:  n,
-					Cause: fmt.Sprintf("failed to get node info [%s]. Err: %v", n.Name, err),
-				}
-			}
-		}
```

Comment:
why do we remove this code?

Reply:
That piece of code is not necessary and affects test when using service type as Load Balancer